routes/routes.py

```diff
@@ -1,12 +1,30 @@
 from flask import Blueprint
-
+from flask import jsonify, request
 router = Blueprint("router", __name__)
 
 @router.route("/check")
 def check():
     return "Congratulations! Your app works. :)"
 
-@router.route("/add", methods=["POST"])
+@router.route('/add', methods = ['POST'])
 def add():
-    # Add logic here
-    return
\ No newline at end of file
+
+    result = {
+        'result' : None,
+        'meta' : None
+    }
+    data_in = request.json['data']
+    if not data_in:
+        result['meta'] = "no json data was provided."
+    values = []
+    for i in data_in:
+        values.append(data_in[i])
+        result['result'] = sum(values)
+    return jsonify(result)
+
+# @ router.route('/subtract', methods = ['POST'])
```

Comment:
Please remove unnecessary comments on your code

Reply:
They have all been removed in my latest commit. Sorry for this.